modules/serverless-creating-subscriptions-ODC.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-creating-subscriptions-ODC_{context}"]
+= Connecting a channel to a sink using subscriptions in the Developer perspective
+
+.Prerequisites
+To create subscriptions using the *Developer* perspective ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created an event sink, such as a Knative service, and a channel.
+
+.Procedure
+. Navigate to the *Topology* page.
+. Create a subscription using one of the following methods:
+.. Hover over the channel that you want to create a subscription for, and drag the arrow. The option is displayed to *Add Subscription*.
```

Comment:
Suggest: The *Add Subscription* option is displayed.

Reply:
Thanks @Preeticp - I'll address this when I'm addressing any feedback from the OpenShift peer reviews to save having multiple small commits :slightly_smiling_face: